flink/src/test/java/org/apache/iceberg/flink/data/RandomData.java

```diff
@@ -88,6 +93,34 @@ public Row next() {
     };
   }
 
+  private static Iterable<RowData> generateRowData(Schema schema, int numRecords,
+      Supplier<RandomRowGenerator> supplier) {
+    DataStructureConverter<Object, Object> converter =
+        DataStructureConverters.getConverter(TypeConversions.fromLogicalToDataType(FlinkSchemaUtil.convert(schema)));
```

Comment:
Here  we may need to call `converter.open(RandomData.class.getClassLoader())` to initialize the converter ?

Reply:
Yes, we can, only `StructuredObjectConverter` implements `open`, but now, Flink not support structure type. (It is not RowType).
I'll revert this method in `RandomData`, it is not be used.